docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -58,6 +57,14 @@ want to be notified earlier about a potential anomaly even if it might be a
 false positive. If you want to get notified only about anomalies of fully 
 processed buckets, do not include interim results.
 
+You can also configure advanced settings. _Lookback interval_ sets an interval 
+that is used to query previous anomalies during each condition check. _Number of 
+latest buckets_ sets how many buckets to check to obtain the highest anomaly 
+from all the anomalies that are found during the _Lookback interval_. An alert 
+is created based on the anomaly with the highest anomaly score from the most 
+anomalous bucket. _Lookback interval_ is derived from the bucket span of the job 
+and the query delay of the {dfeed} by default.
+
```

Comment:
1. I think first you should describe the lookback interval and after that the number of buckets. At the moment they are mixed 
2. Worth mentioning that we don't recommend setting the lookback interval lower than the default one because anomalies can be missed potentially. 
3. do we want to provide more details about the formula used to derived the lookback interval?

Reply:
@darnautov 
1. and 2. are addressed via be39a48
3. I don't think we should include such a low-level detail in the documentation.